pkg/printers/internalversion/describe.go

```diff
@@ -3273,7 +3273,11 @@ func printNetworkPolicySpecIngressFrom(npirs []networking.NetworkPolicyIngressRu
 		} else {
 			for _, from := range npir.From {
 				w.Write(LEVEL_0, "%s", initialIndent)
-				if from.PodSelector != nil {
+				if from.PodSelector != nil && from.NamespaceSelector != nil {
+					w.Write(LEVEL_0, "%s: %s\n         %s: %s\n",
+						"From NamespaceSelector", metav1.FormatLabelSelector(from.NamespaceSelector),
+						"PodSelector", metav1.FormatLabelSelector(from.PodSelector))
```

Comment:
From sig-network's POV, the output is correct, but I don't know if the code style is correct from sig-cli's POV. (ie, the use of spaces to align the two rows)

Reply:
Thanks for reviewing. : )